IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -0,0 +1,317 @@
+                    temp_list = entry.split(":")
+                    stain.add(temp_list[0])
+                    stain_target.add(temp_list[1])
+
+            except (ValueError, KeyError):
+                stain = "Not listed"
+                stain_target = "Not listed"
+
+            # Get cell line
+            try:
+                cell_line_index = [x["ns"] for x in annotations].index(
+                    'openmicroscopy.org/mapr/cell_line')
+                cell_line = {x[0]: x[1] for x in
+                             annotations[cell_line_index]["values"]}["Cell Line"]
+            except (ValueError, KeyError):
+                cell_line = "Not listed"
+
+            # Get strain of cell line
+            try:
+                strain = {x[0]: x[1] for x in
+                          annotations[bulk_index]["values"]}["Strain"]
+            except (ValueError, KeyError):
+                strain = "Not listed"
+
+            # Get gene identification accession code
+            try:
+                gene_identifier_index = int(
+                    [x["ns"] for x in annotations].index(
+                        'openmicroscopy.org/mapr/gene'))
+                gene_identifier = {x[0]: x[1] for x in annotations[gene_identifier_index]["values"]}[
+                    "Gene Identifier"]
+            except (ValueError, KeyError):
+                gene_identifier = "Not Listed"
+
+            # Get phenotype identification accession code
+            try:
+                phenotype_identifier_index = int(
+                    [x["ns"] for x in annotations].index('openmicroscopy.org/mapr/phenotype'))
+                phenotype_identifier = {x[0]: x[1] for x in annotations[phenotype_identifier_index]["values"]}[
+                    "Phenotype Term Accession"]
+            except (ValueError, KeyError):
+                phenotype_identifier = "Not Listed"
+
+            # Build results
+            plate_results.append([
+                screen_id,
+                plate,
+                plate_name,
+                id,
+                cell_line,
+                strain,
+                gene_identifier,
+                phenotype_identifier,
+                stain,
+                stain_target,
+                pixel_size_x,
+                pixel_size_y
+            ])
+
+    # Output results
+    plate_results_df = pd.DataFrame(
+        plate_results,
+        columns=[
+            "screen_id",
+            "plate_id",
+            "plate_name",
+            "image_id",
+            "cell_line",
+            "strain",
+            "gene_identifier",
+            "phenotype_identifier",
+            "stain",
+            "stain_target",
+            "pixel_size_x",
+            "pixel_size_y"
+        ]
+    )
+    return plate_results_df
+
+
+# In[3]:
+
+
+data_dir = pathlib.Path("data")
+
+
+# In[4]:
+
+
+# Load IDR ids
+data_dir = pathlib.Path(
+    "~/Documents/publicly-available-microscopy-data/IDR/data")
```

Comment:
relative, not absolute paths (maybe this is an artifact of not re-running nbconvert)

Reply:
I regenerated the converted notebook files to include changes made during this PR using `nbconvert` command.

Added in d165a301eb7222601fff5ce52254c2999a800f47.